puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/debian/stage_packages.sh

```diff
@@ -13,13 +13,15 @@ DEB_PATH=./debian/${os}/build-${project}
 echo "scratch build: uploading to stagingdeb/${os}/${repoowner}-${version}"
 export RSYNC_RSH="ssh -i /home/jenkins/workspace/staging_key/rsync_freightstage_key"
 USER=freightstage
-HOST=web02
+HOSTS=web02.rackspace web01.osuosl
 COMPONENT=${repoowner}-${version}
 
-# The path is important, as freight_rsync (which is run on the web node for incoming
-# transfers) will parse the path to figure out the repo to send debs to.
-TARGET_PATH="${USER}@${HOST}.theforeman.org:rsync_cache/${os}/${COMPONENT}/"
+for HOST in HOSTS; do
+  # The path is important, as freight_rsync (which is run on the web node for incoming
+  # transfers) will parse the path to figure out the repo to send debs to.
+  TARGET_PATH="${USER}@${HOST}.theforeman.org:rsync_cache/${os}/${COMPONENT}/"
 
-if ls $DEB_PATH/*deb >/dev/null 2>&1; then
-  /usr/bin/rsync -avPx $DEB_PATH/*deb $TARGET_PATH
-fi
+  if ls $DEB_PATH/*deb >/dev/null 2>&1; then
```

Comment:
Perhaps this if should be outside of the for loop? Same applies to other scripts.

Reply:
technically yes, but I didn't want to also change the logic flow of the script, and the loop needs to be outside of `TARGET_PATH` as that contains `HOST`.

Reply:
We should really rewrite these to Jenkins pipelines. But I'm ok with merging it as is.

Reply:
Absolutely!